pkg/packagekit/updates.jsx

```diff
@@ -56,18 +56,43 @@ const PK_STATUS_STRINGS = {
     14: _("Verifying"),
 }
 
+const I_TRANS = "org.freedesktop.PackageKit.Transaction";
```

Comment:
Since this is used pretty rarely, I wonder if we shouldn't name this variable more descriptively. Also, why should it be in caps?